tests/cugraph/cugraph-ops/test_cugraph_relgraphconv.py

```diff
@@ -18,9 +22,10 @@ def generate_graph():
     g.edata[dgl.ETYPE] = torch.randint(num_rels, (g.num_edges(),))
     return g
 
+@pytest.mark.parametrize('use_long', use_longs)
 @pytest.mark.parametrize('max_in_degree', max_in_degrees)
 @pytest.mark.parametrize("regularizer", regularizers)
```

Comment:
Just for a sanity check, the following test is completely disabled. Is this intended?